tests/manage/z_cluster/tier4_helpers.py

```diff
@@ -0,0 +1,202 @@
+import logging
```

Comment:
suggest not having an helper file specifically for tier4 and under the z_cluster dir. 
Please incorporate the function into existing util file like tests/helpers.py